openmc/data/multipole.py

```diff
@@ -434,7 +433,10 @@ def from_hdf5(cls, group_or_filename):
             group = group_or_filename
         else:
             h5file = h5py.File(group_or_filename, 'r')
-            version = h5file['version'].value[0].decode()
+            try:
```

Comment:
This is odd.  Why is this try/except block necessary?  Is H5PY interpreting it differently depending on the version?  If so, it might be worthwhile to only put that exception in the except block so that other errors can properly throw.